README.org

```diff
@@ -46,6 +46,11 @@ If you use ~use-package~, here is the recipe for installing it.
     :bind (:map dired-mode-map ("C-o" . 'casual-dired-tmenu)))
 #+end_src
 
+In addition, you may wish to directly sort the Dired buffer by
```

Comment:
One more thing, please make the paragraph have no line breaks ("\n"). Thanks!

Reply:
I'm sorry about that. I knew you wanted a single line but my fingers automatically press `M-q` all the times and I didn't pay attention when reviewing the changes.